st2common/bin/st2-self-check

```diff
@@ -89,9 +89,9 @@ if [ ${EXIT_CODE} -ne 0 ]; then
 fi
 
 echo "Copying asserts, fixtures, tests and examples packs."
-cp -R st2tests/packs/* /opt/stackstorm/packs/
-cp -Rf /usr/share/doc/st2/examples /opt/stackstorm/packs/
-
+chown -R root:st2packs st2tests/packs/
+cp -R --preserve st2tests/packs/* /opt/stackstorm/packs/
```

Comment:
I was testing on `EL6`.
So yeah, after verifying on `EL6` I don't expect anything worse to happen :stuck_out_tongue: